src/main.h

```diff
@@ -174,7 +177,8 @@ void UnregisterNodeSignals(CNodeSignals& nodeSignals);
  * @param[out]  dbp     If pblock is stored to disk (or already there), this will be set to its location.
  * @return True if state.IsValid()
  */
-bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp);
+bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp,
+libzcash::ProofVerifier& verifier = disabledVerifier);
```

Comment:
Ohh but OTOH most methods that use disabled verifier will all have to define it explicitly, so there's the efficiency danger of multiply verifying jointsplits, and the extra code of defining many disabled verifiers